freqtrade/rpc/telegram.py

```diff
@@ -31,7 +31,7 @@ def authorized_only(command_handler: Callable[..., None]) -> Callable[..., Any]:
     """
     def wrapper(self, *args, **kwargs):
         """ Decorator logic """
-        update = kwargs.get('update') or args[1]
+        update = kwargs.get('update') or args[0]
```

Comment:
Did not get it yet, while looking at first glance...

(No need to answer if it's long and includes excurces into the telegram bot API and their differences)

I just marked the part I did not get during quick review of the changes.

Reply:
Sorry, didn't remember that you didn't use telegram...

this looks odd at first, yes - it's a wrapper around all other functions - and since the headers changed from having update as 2nd argument `[1]` to the first argument `[0]` this change was required.

It's directly caused by the interface change of telegram ... (sample below)

``` python
def _xxx(self, bot: Bot, update: Update) -> None:
# to
 def _xxx(self, update: Update, context: CallbackContext) -> None:
```